modules/clients/src/test/java/org/apache/ignite/internal/processors/rest/RestMemcacheProtocolSelfTest.java

```diff
@@ -185,8 +185,8 @@ public void testReplace() throws Exception {
      */
     @Test
     public void testMetrics() throws Exception {
-        grid().cache(DEFAULT_CACHE_NAME).localMxBean().clear();
-        grid().cache(CACHE_NAME).localMxBean().clear();
+        grid().cache(DEFAULT_CACHE_NAME).clearStatistics();
```

Comment:
Why do we need new method `clearStatistics`?
AFAICU clear method comes from `javax.cache.management.CacheStatisticsMXBean`
Can we just invoke method from MX bean instance as we do now?

Reply:
> Why do we need new method `clearStatistics`? AFAICU clear method comes from `javax.cache.management.CacheStatisticsMXBean` 

This method calls GridCacheProcessor#clearStatistics(). It isn't new. I brought other call in the tests to keep the conception of mxbean removal.
We keep them now only to support JCache spec. I suggest not to use JCache mxbean interfaces any more even in the tests. WDYT?

Reply:
> This method calls GridCacheProcessor#clearStatistics().

How to do the same with public API?

Reply:
> > This method calls GridCacheProcessor#clearStatistics().
> 
> How to do the same with public API?

You always could clear cache statistics by calling Ignite cache API and by calling mx bean. Regarding to the IEP, we assume mx bean api pbsolete. Why using It again? Consider we wouldn't have to keep mxbean to support any spec like JCache spec. There would be no mxbean and no old mxbean#clear() method.

Reply:
> You always could clear cache statistics by calling Ignite cache API

What API, exactly? Why in tests you are using private API?

Reply:
> > You always could clear cache statistics by calling Ignite cache API
> 
> What API, exactly? Why in tests you are using private API?

It is not private. It uses IgniteCache#clearStatistics(). It is a public API.